package.json

```diff
@@ -39,5 +40,10 @@
     "rimraf": "^2.5.2",
     "standard": "^10.0.2",
     "temporary": "^0.0.8"
+  },
+  "standard": {
+    "ignore": [
+      "app/client/lib"
```

Comment:
Ignoring this for now, but something we might want to address as well? I'm unsure how the lens code is built/managed etc.